src/gui/mainwindow.h

```diff
@@ -88,6 +90,7 @@ private slots:
 	QAction* m_actPaste{ nullptr };
 	QAction* m_actSelectAll{ nullptr };
 	ScrollBar* m_scrollbar{ nullptr };
+	int m_exitStatus = 0;
```

Comment:
nitpick: Initialize with `{}` when possible. The compiler will do more aggressively type-checking for that syntax.

Elsewhere too.